templates/github/.github/workflows/scripts/before_install.sh.j2

```diff
@@ -166,6 +166,8 @@ then
   exit $s
 fi
 
+sed -i "s/DEBUG = False/DEBUG = True/" pulpcore/pulpcore/app/settings.py
```

Comment:
```suggestion
sed -i -e 's/DEBUG = False/DEBUG = True/' pulpcore/pulpcore/app/settings.py
```
Just a matter of personal style, but i like to explicit about the "expression" with sed. Also using single quotes usually keeps you from close to crazy shell escaping (a good habit even if not applicable here).

Reply:
Needless to say, this is exactly what it is labelled: a suggestion